invenio_accounts/version.py

```diff
@@ -28,4 +28,6 @@
 and parsed by ``setup.py``.
 """
 
-__version__ = "0.2.1.dev20151005"
+from __future__ import absolute_import, print_function
```

Comment:
It's to standardise every python file to always have the same ``__future__`` imports. Not something I feel strongly about, but we should fix it upstream in cookiecutter.